src/Tools/Source/RunTests/Options.cs

```diff
@@ -138,10 +138,24 @@ internal class Options
             Architecture = architecture;
         }
 
+        private static string GetArchitecture() => RuntimeInformation.ProcessArchitecture switch
+        {
+            System.Runtime.InteropServices.Architecture.Arm => "arm",
+            System.Runtime.InteropServices.Architecture.Arm64 => "arm64",
+            System.Runtime.InteropServices.Architecture.Armv6 => "armv6",
+            System.Runtime.InteropServices.Architecture.LoongArch64 => "loongarch64",
+            System.Runtime.InteropServices.Architecture.Ppc64le => "ppc64le",
+            System.Runtime.InteropServices.Architecture.S390x => "s390x",
+            System.Runtime.InteropServices.Architecture.Wasm => "wasm",
+            System.Runtime.InteropServices.Architecture.X86 => "x86",
```

Comment:
I was thinking about this a bit: do we want to handle `x86` at all? IIRC there have been some problems in the past where this forced x86 in VisualStudio, even on x64 machines.

Reply:
I would keep it there for completeness but this is only called on Linux paths and I don't think we really test x86 linux to my knowldege.